dist/types/system/SetStartingDocumentNumberRequest.js

```diff
@@ -0,0 +1,2 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Why do we have empty js-modules in a `types` directory? Looks like something is misconfigured 🤔